projects/extension/src/background/ConnectionManager.ts

```diff
@@ -49,27 +49,26 @@ export class ConnectionManager extends (EventEmitter as { new(): StateEmitter })
    *
    * @returns a list of the networks that are currently connected
    */
-  get registeredClients(): string[] {
-    return this.#networks.map(s => s.name);
+  get registeredClients(): unknown[] {
```

Comment:
Why is this `unknown`?  We should know the type of this

Reply:
Forgot to check linter - CI grab all those and Im fixing them atm